routes/web.php

```diff
@@ -54,4 +54,6 @@
         $router->post('add_nick_name','NicknameController@addNickName');
     });
 
+    $router->post('/pageAdsListing','AdsController@pageAdsListing');
+    $router->post('/pageImagesListing','ImageController@pageImagesListing');
```

Comment:
Please follow the correct url convention. It is not matching with what are using